src/layout/general_settings.rs

```diff
@@ -48,6 +49,15 @@ pub struct GeneralSettings {
     pub separators_color: Color,
     /// The text color to use for text that doesn't specify its own color.
     pub text_color: Color,
+    /// Specifies the display accuracy of split times.
+    pub split_time_accuracy: Accuracy,
+    /// Specifies the display accuracy of segment times.
+    pub segment_time_accuracy: Accuracy,
+    /// Specifies the display accuracy of delta times.
+    pub delta_time_accuracy: Accuracy,
+    /// Whether to drop the fractional part of a delta time once it goes past
+    /// one minute.
+    pub delta_drop_decimals: bool,
```

Comment:
Yeah these should be moved to the settings of the Splits component: https://github.com/LiveSplit/livesplit-core/blob/master/src/component/splits/mod.rs#L48

You can pass these settings to the columns function as a parameter.

These settings then also need to be exposed in the layout editor via these two functions: https://github.com/LiveSplit/livesplit-core/blob/master/src/component/splits/mod.rs#L434-L552

Reply:
Oh right, thank you! I'll try to get this done tomorrow. Would you recommend passing the whole of the split settings to the column update, or just those relevant parts? dunno how tightly coupled columns and splits are allowed/supposed to be

Reply:
Passing the whole thing would keep the argument list smaller (and thus faster), so I'd go with that.